### PR TITLE
added all assignments

### PR DESCRIPTION
**Pull request by vishnuvkaimal**
Fixes #

## Proposed Changes

-
-
-

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/51"></a>

### CODE REVIEW

File: Vishnu/Assignment1.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Merlo J.Pusey</title>
+</head>
+<body>
+<h2><b>Merlo J. Pusey</b></h2>
+<hr>
+<p><i>Wikipedia, the free encyclopedia</i></p>
```
Comment (VinayakTekade) [inline Vishnu/Assignment1.html]: Use `cite` instead of `i`

File: Vishnu/Assignment1.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Merlo J.Pusey</title>
+</head>
+<body>
+<h2><b>Merlo J. Pusey</b></h2>
```
Comment (VinayakTekade) [inline Vishnu/Assignment1.html:10]: Use `strong` instead of `b`

File: Vishnu/Assignment1.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+<h3>Books</h3>
+<hr>
+<p> 
+    <ul>
+    <li>Supreme Court Crisis(<i>Macmillan, 1937</i>)</li>
+    <li>Big Government: Can We Control It? (<i>Harper, 1945</i>)</li>
+    <li>Charles Evans Hughes (<i>2 vols., Macmillan, 1951</i>)</li>
+    <li>Eisenhower, the President (<i>Macmillan, 1956</i>)</li>
+    </ul>
+</p>
```
Comment (VinayakTekade) [inline Vishnu/Assignment1.html]: `p` is not needed. Use `div` instead.

File: Vishnu/Assignment1.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<p><img src="https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg" alt=""> </p>
+<p><b>Merlo John Pusey</b> (February 3, 1902, in Woodruff, Utah – November 22, 1985, in Washington, D.C.)<sup>[1][2]</sup> was an <u>American biographer</u> and <u>editorial writer</u>. He won the 1952 Pulitzer Prize for Biography or Autobiography
+and the 1952 Bancroft Prize for his 1951 biography of U.S. Chief Justice Charles Evans Hughes.</p>
+
+<p>Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the college newspaper. He later became
+a reporter and assistant city editor at The Deseret News in Salt Lake City.<sup>[3]</sup></p>
+
+<p><s>Pusey</s> worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946, continuing to contribute occasional pieces until about two years before his death.</p>
+
+<P><b><Contents/b> </P>
```
Comment (VinayakTekade) [inline Vishnu/Assignment1.html]: Use `strong` instead of  `b`

File: Vishnu/Assignment3.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+<form action="" method="post">
+<h1>Register</h1>
+<p>
+    First Name <input type="text" name="FirstName" required>
```
Comment (VinayakTekade) [inline Vishnu/Assignment3.html]:
```suggestion
    First Name <input type="text" name="FirstName" required autocomplete="given-name">
```

This an example of how to use `autocomplete` , use this same format for the other fields marked as `autocomplete` missing

File: Vishnu/Assignment3.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    Last Name <input type="text" name="LastName" required>
```
Comment (VinayakTekade) [inline Vishnu/Assignment3.html]: Missing `autocomplete`

File: Vishnu/Assignment3.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+</p>
+<p>
+    Gender: Male: <input type="radio" name="Gender" value="male"/>
+    Female: <input type="radio" name="Gender" value="female"/>
+    Others: <input type="radio" name="Gender" value="others"/>
+</p>
+<p>
+Email: <input type="email" name="Email" id="email" required>
```
Comment (VinayakTekade) [inline Vishnu/Assignment3.html]: Missing `autocomplete`

File: Vishnu/Assignment3.html [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+Password: <input type="password" name="pwd" min="5" max="15" required>
```
Comment (VinayakTekade) [inline Vishnu/Assignment3.html]: Missing `autocomplete`

File: Vishnu/Assignment3.html [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+Birthsday <input type="date" name="birthday">
```
Comment (VinayakTekade) [inline Vishnu/Assignment3.html]: Missing `autocomplete`

**Review by VinayakTekade — DISMISSED** [review]

**Review by VinayakTekade — APPROVED** [review]
All the changes made are correct. Nice job.

File: Vishnu/Assignment2.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    <thead>
+        <tr>
+            <td><B>Image</B></td>
+            <td><B>Name</B></td>
+            <td><B>Type</B></td>
+            <td><B>Evolves Into</B></td>
+        </tr>
+    </thead>
+    <tr>
+        <td><img src="http://img4.wikia.nocookie.net/__cb20140328190757/pokemon/images/thumb/2/21/001Bulbasaur.png/200px-001Bulbasaur.png" alt=""></td>
```
Comment (aravindvnair99) [inline Vishnu/Assignment2.html:21]: Switch to `HTTPS` for next pull request.

File: Vishnu/Assignment2.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+        <td>Bulbasaur</td>
+        <td>Grass/Poison</td>
+        <td><a href="http://pokemon.wikia.com/wiki/Ivysaur">Ivysaur</a></td>
+    </tr>
+
+    <tr>
+        <td><img src="http://img4.wikia.nocookie.net/__cb20140724195345/pokemon/images/thumb/7/73/004Charmander.png/200px-004Charmander.png" alt=""></td>
```
Comment (aravindvnair99) [inline Vishnu/Assignment2.html:28]: Switch to `HTTPS` for next pull request.

File: Vishnu/Assignment2.html [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+        <td>Charmander</td>
+        <td>Fire</td>
+        <td><a href="http://pokemon.wikia.com/wiki/Charmeleon">Charmeleon</a></td>
+    </tr>
+    <tr>
+        <td><img src="http://img1.wikia.nocookie.net/__cb20140328191525/pokemon/images/thumb/3/39/007Squirtle.png/200px-007Squirtle.png" alt=""></td>
```
Comment (aravindvnair99) [inline Vishnu/Assignment2.html:34]: Switch to `HTTPS` for next pull request.

File: Vishnu/Assignment2.html [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+        <td>Squirtle</td>
+        <td>Water</td>
+        <td><a href="http://pokemon.wikia.com/wiki/Wartortle">Wartortle</a></td>
```
Comment (aravindvnair99) [inline Vishnu/Assignment2.html:37]: Switch to `HTTPS` for next pull request.

File: Vishnu/Assignment2.html [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+        <td>Bulbasaur</td>
+        <td>Grass/Poison</td>
+        <td><a href="http://pokemon.wikia.com/wiki/Ivysaur">Ivysaur</a></td>
+    </tr>
+
+    <tr>
+        <td><img src="http://img4.wikia.nocookie.net/__cb20140724195345/pokemon/images/thumb/7/73/004Charmander.png/200px-004Charmander.png" alt=""></td>
+        <td>Charmander</td>
+        <td>Fire</td>
+        <td><a href="http://pokemon.wikia.com/wiki/Charmeleon">Charmeleon</a></td>
```
Comment (aravindvnair99) [inline Vishnu/Assignment2.html:31]: Switch to `HTTPS` for next pull request.

File: Vishnu/Assignment2.html [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+            <td><B>Name</B></td>
+            <td><B>Type</B></td>
+            <td><B>Evolves Into</B></td>
+        </tr>
+    </thead>
+    <tr>
+        <td><img src="http://img4.wikia.nocookie.net/__cb20140328190757/pokemon/images/thumb/2/21/001Bulbasaur.png/200px-001Bulbasaur.png" alt=""></td>
+        <td>Bulbasaur</td>
+        <td>Grass/Poison</td>
+        <td><a href="http://pokemon.wikia.com/wiki/Ivysaur">Ivysaur</a></td>
```
Comment (aravindvnair99) [inline Vishnu/Assignment2.html:24]: Switch to `HTTPS` for next pull request.

File: Vishnu/Assignment3.html [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+<form action="" method="post">
+<h1>Register</h1>
+<p>
+    First Name <input type="text" name="FirstName" required autocomplete="given-name">
+    Last Name <input type="text" name="LastName" required autocomplete="family-name">
+</p>
+<p>
+    Gender: Male: <input type="radio" name="Gender" value="male"/>
+    Female: <input type="radio" name="Gender" value="female"/>
+    Others: <input type="radio" name="Gender" value="others"/>
```
Comment (aravindvnair99) [inline Vishnu/Assignment3.html:19]: Add `label`.

File: Vishnu/Assignment3.html [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+</p>
+<p>
+Email: <input type="email" name="Email" id="email" required autocomplete="email">
+Password: <input type="password" name="pwd" min="5" max="15" required autocomplete="current-password">
```
Comment (aravindvnair99) [inline Vishnu/Assignment3.html:23]: Wrong `autocomplete` attribute. Should be for a registration form, not login. So should be new and not current as there is no current.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by aravindvnair99** [comment]
@vishnuvkaimal Update your PR description. Use proper GFM syntax henceforth with proper escape characters. Refer this: [GFM - GitHub Guides](https://guides.github.com/features/mastering-markdown/)

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@vishnuvkaimal Make the changes intimated to you on the group.

**Comment by aravindvnair99** [comment]
@vishnuvkaimal Closing this pull request due to inactivity. You can send in the right one as a new pull request. Have a look at the video I sent on the group.

**Pull request closed without merging**